src/NuGet.Core/NuGet.PackageManagement/Projects/PackagesConfigNuGetProject.cs

```diff
@@ -60,11 +58,9 @@ public PackagesConfigNuGetProject(string folderPath, Dictionary<string, object>
                 throw new ArgumentNullException(nameof(folderPath));
             }
 
-            TargetFramework = GetMetadata<NuGetFramework>(NuGetProjectMetadataKeys.TargetFramework);
-
             PackagesConfigPath = Path.Combine(folderPath, "packages.config");
 
-            var projectName = GetMetadata<string>(NuGetProjectMetadataKeys.Name);
+            var projectName = Metadata[NuGetProjectMetadataKeys.Name];
```

Comment:
Otherwise it will anyway trigger initializing metadata right into the constructor, there won't be any sense of this code change then... Kind of a heck, i could prevent it but then it would a little bigger change. Since we might not be investing here too much and transitioning to PackageReference, so i thought to keep it that way.